ocs_ci/deployment/deployment.py

```diff
@@ -241,6 +242,95 @@ def do_deploy_rdr(self):
             deploy_dr = MultiClusterDROperatorsDeploy(dr_conf)
             deploy_dr.deploy()
 
+    def do_deploy_lvmo(self):
```

Comment:
For keeping the consistency with other deploy class method, please change to `deploy_lvmo`

Reply:
But I've kept consistency:
        self.do_deploy_lvmo()
        self.do_deploy_submariner()
        self.do_deploy_ocs()
        self.do_deploy_rdr()